src/main/java/com/plivo/api/models/base/BaseRequest.java

```diff
@@ -92,7 +92,7 @@ protected void handleResponse(Response response) throws PlivoRestException, IOEx
         throw new ServerException(response.errorBody().string());
     }
 
-    if (this instanceof Deleter && responseCode != 204) {
+    if (this instanceof Deleter && !(responseCode == 204 || responseCode == 200)) {
```

Comment:
Does powerpack return contents in body for DELETE requests? Why not make powerpack return 204 if it doesn't return a body?

Reply:
Response has body, If not put 200 status code on check here, next line will look for error and throw the exception